processes/security_team_membership_policy.md

```diff
@@ -0,0 +1,27 @@
+# Node.js Security Team Membership Policy
+
+The Node.js Security Team, namely the @nodejs/security team on Github, has
+access to security-sensitive issues and patches that aren't approriate for
+public availability.
+
+> Note: Due to implementation details and a bit of human error, the list of
+> people with access to the security patches isn't 100% in line with the
+> @nodejs/security membership. See the [membership list](./security_team_members.md)
+> for details. This situation is intended to be corrected in the near future.
+
+This isn't to be confused with the security triage team, which is currently
+defined by the [`security@nodejs.org` email alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json).
+Inclusion in that group is on a volunteer basis, upon approval by the Technical
+Steering Committee (ie. @nodejs/TSC).
+
+The policy for inclusion on the @nodejs/security is as follows:
+
+1. All members of the Technical Steering Committee (i.e. @nodejs/TSC) are
+   members of @nodejs/security.
+2. On a case-by-case basis, individuals outside the Technical Steering Committee
+   are invited by the TSC to join @nodejs/security so that their expertise can
+   be applied to an issue or patch. Once their assistance is no longer required
+   for a particular issue or patch, they are removed from the team.
+3. Members of the [release team](https://github.com/nodejs/node#release-team)
+   who are not also members of the TSC are also members of @nodejs/security, as
+   they need access to the patches in order to produce releases.
```

Comment:
That policy would remove me and a couple of others (e.g. @dougwilson, @sam-github) that should stay on.

(Maybe Doug could be added on a case-by-case basis for express-related issues but I'm conceited enough to think that doesn't apply to me.)

Reply:
I was originally added bexause I asked mainly because I was interested in being aware of and/or helping with http related ones that came up (since Express is closely tied to the http server in Node).

But I am also not going to ask the group to break their policy to keep me here just because I am currently here :) I am not truely part of the team, mearly an outside observer. Ideally if there is a way to add outside folks at an issue level instead of all or nothing that may help, but I believe that is out of scope here. 

If I am removed, something to consider for the future :)

Reply:
@bnoordhuis You'd be included via membership in the triage team. (I had forgotten to add that. It's there now.)

Reply:
Agreed all members of the triage team should also have access.